scene-layer/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        applicationId "com.esri.arcgisruntime.sample.displayscene"
```

Comment:
update `applicationId` to be specific to this sample, e.g. `com.esri.arcgisruntime.sample.scenelayer`.